iceoryx_utils/source/posix_wrapper/shared_memory_object/memory_map.cpp

```diff
@@ -44,36 +44,26 @@ MemoryMap::MemoryMap(const void* baseAddressHint,
     }
     // PRQA S 3066 1 # incompatibility with POSIX definition of mmap
```

Comment:
I have no idea since we do not have access to PRQA S 3066 1. @marthtz what does it state?

Maybe we should make the rule that all those QAC++ exception statements must be accessible by all developers otherwise a fellow developer has no idea what it means and if it has to be adapted when the code is refactored like in this case.

Reply:
@dkroenke is working on a mapping table Axivion warnings <-> QAC warnings